capstone/src/main/java/com/google/univiz/api/SearchData.java

```diff
@@ -1,4 +1,15 @@
 package com.google.univiz.api;
 
+import com.google.auto.value.AutoValue;
+
 /** SearchData represents the college data specific to the Search feature */
-public class SearchData {}
+@AutoValue
+public abstract class SearchData {
+  public static SearchData create(String collegeName, int collegeId) {
+    return new AutoValue_SearchData(collegeName, collegeId);
+  }
```

Comment:
```suggestion

```

Reply:
Hi, as per my previous comment, I'm still wondering why I need to delete this? I don't doubt that this is a correct way to do it, but I don't understand why it needs to be done :)

Reply:
I think you want to leave this in, since you'll create these instances from `SearchResourceImpl`